css/_variables.scss

```diff
@@ -195,12 +195,12 @@ $unsupportedDesktopBrowserTextFontSize: 21px;
 /**
  * The size of the default watermark.
  */
-$watermarkWidth: 230px;
-$watermarkHeight: 55px;
+$watermarkWidth: 366px;
+$watermarkHeight: 90px;
 $watermarkTop: 32px;
 
-$welcomePageWatermarkWidth: 186px;
-$welcomePageWatermarkHeight: 74px;
+$welcomePageWatermarkWidth: 366px;
+$welcomePageWatermarkHeight: 90px;
```

Comment:
for me, its too big, isn't it?

Reply:
This was the designed size, however i made it to scale on smaller resolutions